nemo/collections/tts/torch/readme.md

```diff
@@ -49,14 +49,3 @@ for batch in tqdm(dataloader, total=len(dataloader)):
 pitch_tensor = torch.cat(pitch_list)
```

Comment:
The get_cmudict.sh script is redundant. We have the entire dict in our repo. Let's just delete that file.